extra_deps patch generated by KarLS Builder Infrastructure
Added extra_deps.lst
diff -ruN --no-dereference src/grub-2.12.orig/grub-2.12/grub-core/extra_deps.lst src/grub-2.12/grub-core/extra_deps.lst
--- src/grub-2.12.orig/grub-2.12/grub-core/extra_deps.lst	1970-01-01 01:00:00.000000000 +0100
+++ src/grub-2.12/grub-core/extra_deps.lst	2024-03-16 23:56:44.327701778 +0100
@@ -0,0 +1 @@
+depends bli part_gpt
